nuget/helpers/lib/NuGetUpdater/.editorconfig

```diff
@@ -131,7 +133,7 @@ csharp_new_line_before_catch = true
 csharp_new_line_before_else = true
 csharp_new_line_before_finally = true
 csharp_new_line_before_members_in_anonymous_types = true
-csharp_new_line_before_members_in_object_initializers = true
+csharp_new_line_before_members_in_object_initializers = false
```

Comment:
We shouldn't change the existing configuration. Assuming the code is code we wrote, we would want to fix it up appropriately. If it is code we borrowed, then we may need to exclude it from style enforcement.

Reply:
This configuration with `true` actually caused this:
https://github.com/dependabot/dependabot-core/pull/8930/commits/a3b67f31207c7331022492b319b208044a7c8a40

So you need to choose, collection initializers on a single line or config change :)

Reply:
Running dotnet-format locally against the code in main does not convert object initializers from multi-line to single-line.

I can see from the commit history that the 'Apply Rider rules' commit is what changed the initializers to single-line. https://github.com/dependabot/dependabot-core/pull/8930/commits/ccd9c046a9162cd61bc599899cc0ec4b82b73ff2

Reply:
When I set it to `true` and do reformat in Rider, it changes to the before of the commit https://github.com/dependabot/dependabot-core/commit/a3b67f31207c7331022492b319b208044a7c8a40 (this commit reverts some changes from 'Apply Rider rules').

Since dotnet-format does not do anything with either `true` or `false` it implies to me that it does not understand this configuration.

Reply:
To be fair these are array initializers which are being converted to single-line, so this option wouldn't apply here.

Reply:
That looks like a bug in the Rider formatter. That code is clearly an array initializer. Configuration for object initializers should have no bearing on how it is formatted. Here are the docs for how this setting should affect formatting https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options#csharp_new_line_before_members_in_object_initializers

Reply:
Since the default is `true` for the Roslyn formatter used by dotnet-format, you could try commenting the line out and seeing if Rider behaves differently with the configuration missing.